.github/workflows/compatibility.yml

```diff
@@ -24,8 +24,8 @@ jobs:
     - name: Checkout TiDB
       uses: actions/checkout@v2
       with:
-        repository: JmPotato/tidb
-        ref: update_pd_client
+        repository: crazycs520/tidb
```

Comment:
why our own repo here? not offical pingcap/tidb? @cfzjywxk

Reply:
This is to pass the CI test

Reply:
After https://github.com/pingcap/tidb/pull/58420 merged, https://github.com/tikv/client-go/pull/1540 will recover ci reference to original tidb.

Reply:
@siddontang 

The TiDB compatibility tests in `client-go` can be removed in the future. These tests make modifying `client-go` very cumbersome after the repository split. In practice, updating the `client-go` dependency in TiDB already serves as a compatibility test.